assets/blocks/conditional-content-block/block.json

```diff
@@ -1,5 +1,7 @@
 {
   "name": "sensei-lms/conditional-content",
+  "title": "Conditional Content",
+  "description": "Content inside this block will be shown to the selected subgroup of users.",
```

Comment:
Should have the title and the description on the index.js file to use the i18n features?